addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -497,17 +497,22 @@ class StudyService extends Service {
   }
 
   async list(requestContext, category, fields = []) {
+    const openDataTagFilters = await this.settings.get('openDataTagFilters').split(',');
     // Get studies allowed for user
     let result = [];
     switch (category) {
       case 'Open Data':
         // Readable by all
-        result = await this._query()
-          .index(this.categoryIndex)
-          .key('category', category)
-          .limit(1000)
-          .projection(fields)
-          .query();
+        result = (
+          await this._query()
+            .index(this.categoryIndex)
+            .key('category', category)
+            .limit(1000)
+            .projection(fields)
+            .query()
+        ).filter(study => {
+          return openDataTagFilters.some(filterTag => study.tags.includes(filterTag));
+        });
```

Comment:
Just curious if this filter can be server side as opposed to client side?

Reply:
We discussed this offline and decided to go with filtering in the lamba function because filtering using filter expression is much more complex and the latency difference would be small.